logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -332,10 +332,6 @@ def flush_signal_received?
 
       private
 
-      def iterating?
-        @iterating
-      end
```

Comment:
True, but people aren't always aware whether the code are in a hot loop or not. I only suggested, its your call, but consider that PH, Colin and Andrew are away and Joao will be shortly (paternity).